ffv1.md

```diff
@@ -201,28 +203,41 @@ Background: a two's complement signed 16-bit signed integer was used for storing
 +---+---+---+---+
 ```
 
-The quantized sample differences L-l, l-tl, tl-t, t-T, t-tr are used as context:
+The quantized sample differences are used as context:
 
-PDF:$$context=Q_{0}[l-tl]+\left|Q_{0}\right|(Q_{1}[tl-t]+\left|Q_{1}\right|(Q_{2}[t-tr]+\left|Q_{2}\right|(Q_{3}[L-l]+\left|Q_{3}\right|Q_{4}[T-t])))$$
+PDF:$$context=Q_{0}[l-tl]+Q_{1}[tl-t]+Q_{2}[t-tr]+Q_{3}[L-l]+Q_{4}[T-t]$$
```

Comment:
I added the `abs()` in the RFC intended it to be a semantic equivalent to the pipes used in the LaTeX version. In the context of the plain text RFC, I thought `abs()` would be more readable than `|a|`. But I am not certain that the pipes in the LaTeX version indicate an absolute value but that is what the mathematical function section implies. Ping to @michaelni.